stack.yaml

```diff
@@ -2,6 +2,7 @@ resolver: nightly-2019-02-04
 
 extra-deps:
   - shellmet-0.0.0
+  - github-0.21
```

Comment:
I personally prefer alphabetic order of dependencies in `extra-deps` 🙂 This way I can locate deps easier.